packages/venia-ui/lib/components/AccountInformationPage/accountInformationPage.gql.js

```diff
@@ -20,18 +20,17 @@ export const CHANGE_CUSTOMER_PASSWORD = gql`
     ) {
         changeCustomerPassword(
             currentPassword: $currentPassword
-            newPassword: $newPassword
+            newPassword: $newPassword # eslint-disable-next-line @graphql-eslint/require-id-when-available
```

Comment:
Only small thing is that the comment placement seems weird.
Could we have something like this?
```
changeCustomerPassword(
    currentPassword: $currentPassword
    newPassword: $newPassword
) # eslint-disable-next-line @graphql-eslint/require-id-when-available
{
    email
}
```

Reply:
I tried, but prettier does not like it.